tools/integration_tests/local_file/local_file_test.go

```diff
@@ -34,7 +34,8 @@ import (
 )
 
 const (
-	testDirName = "LocalFileTest"
+	testDirName                 = "LocalFileTest"
+	testDirOnlyDirForLocalFiles = "testDirOnlyDirForLocalFiles"
```

Comment:
Nit: how about this name? 
```suggestion
	OnlyDirMounted = "OnlyDirMountLocalFiles"
```

Same for other packages also